src/js/apps/globals/nav_app.js

```diff
@@ -78,11 +78,13 @@ const patientsAppWorkflowsNav = new Backbone.Collection([
     event: 'worklist',
     eventArgs: ['owned-by'],
   },
+  {},
```

Comment:
feels dirty but works

Reply:
Rather than add a new model just for spacers, we can add support for a `className` on the existing models.

`NavItemView` would need something like this:
```
  className() {
    const className = 'app-nav__link';

    if (this.model.get('className')) {
      return `${ className } ${ this.model.get('className') }`;
    }

    return className;
  },
```
That will allow us to use the standard spacing classnames.

Reply:
Yeah let's add something like `type: 'spacer'` and check for type where you're checking for text.  And then we the default can be the current links.. don't have to type the rest.

Reply:
Ah yeah.  what @LisaManresa said.  Sounds good.

Reply:
yeah what @LisaManresa said it way better than what I did, I'll go with that